addons/website_sale/views/templates.xml

```diff
@@ -184,8 +184,9 @@
         <form action="/shop/cart/update" method="post" class="oe_product_cart h-100 d-flex"
             t-att-data-publish="product.website_published and 'on' or 'off'"
             itemscope="itemscope" itemtype="http://schema.org/Product">
-
-            <t t-set="product_href" t-value="keep(product.website_url, page=(pager['page']['num'] if pager['page']['num']&gt;1 else None))" />
+            <t t-nocache="update_links">
+                <t t-set="product_href" t-value="keep(product.website_url, page=(pager['page']['num'] if pager['page']['num']&gt;1 else None))" />
+            </t>
```

Comment:
Does this work? The elements using `product_href` are cached so I guess it doesn't change anything.

Reply:
Yes it works